project_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -32,7 +32,7 @@ def onchange_task_id(self):
             self.project_id = self.task_id.project_id.id
 
     def eval_date(self, vals):
-        if 'date_time' in vals and 'date' not in vals:
+        if 'date_time' in vals and vals.get('date_time'):
```

Comment:
I think that `get` is not needed in this case.
`if 'date_time' in vals and vals['date_time']:`

Reply:
Actually removing the first part and leaving the `get` would be functionally equivalent and cleaner:
```
if vals.get('date_time'):
```

Reply:
Well, I wanted to avoid the `None` case, but you're right. It was correct the first time.

Reply:
Ok, I'm going to change it as @lasley says.